cypress/integration/pages/articles/testsForCanonicalOnly.js

```diff
@@ -88,6 +88,28 @@ export const testsThatFollowSmokeTestConfigForCanonicalOnly = ({
             }
           });
         });
+
+        it('plays media when a user clicks play', () => {
+          cy.window().then(win => {
+            const media = getBlockData('video', win.SIMORGH_DATA.pageData);
+            if (media && media.type === 'video') {
+              cy.get(
+                'div[class^="StyledVideoContainer"] img[class^="StyledImg"]',
+              )
+                .click()
+                .should('not.exist')
+                .then(() => {
+                  cy.get('iframe[class^="StyledIframe"]').then($iframe => {
+                    cy.wrap($iframe.prop('contentWindow'))
```

Comment:
Would it help for us to wait for the outer iframe to finish loading and/or bump to finish initialising the player before proceeding?

Reply:
Just realised you've partly addressed this in the description but I wonder if there's more we can/should do.

Reply:
The problem I was finding with `load` events was they were firing before they could be attached. This could be got around by checking the document.readyState before binding it, but also meant returning a promise and lengthening the test chain.

- `its()` is designed for this use-case. See [here](https://docs.cypress.io/api/commands/its.html#Existence).